minicluster/src/main/java/org/apache/accumulo/cluster/AccumuloCluster.java

```diff
@@ -53,14 +52,6 @@
    */
   ServerContext getServerContext();
 
-  /**
-   * Utility method to get a connector to the cluster.
-   *
-   * @deprecated since 2.0.0, replaced by {@link #getAccumuloClient(String, AuthenticationToken)}
-   */
-  Connector getConnector(String user, AuthenticationToken token)
```

Comment:
This is public API. It can't be removed unless deprecated first. Same with the subclass impl of this method in StandaloneAccumuloCluster.

Reply:
You're right. My mistake. I misread o.a.a.cluster (which isn't public API) as or.a.a.minicluster (which is public API).